README.md

```diff
@@ -1,58 +1,119 @@
-# Hi there 👋
-
-This repo serves as the main repo for our Rails Projects. 
-
-
-#### Version
-
-```
-  Ruby 2.7.2
-  Rails 6.0.3.4
-  NodeJS 12.18.3
-  Yarn 1.22.4
-```
-
-#### Installation
-
-Fork this repo:
-![image](https://user-images.githubusercontent.com/9253881/111437757-6c527f00-873e-11eb-8d0a-862e4158f4ab.png)
-
-Then, clone your forked repo to your machine
-```
-  $ git clone git@github.com:<username>/rails-base-project.git <name of your app>
-  $ cd <name of your app>
-```
-
-#### Setup
-
-```
-  $ bundle install
+# Drop that beat app   ![build](https://github.com/lea927/drop-that-beat/actions/workflows/rspec.yml/badge.svg)
+
+
+![Drop that beat app](https://user-images.githubusercontent.com/79251819/136054424-c82d19df-477b-4dbc-8f23-0185d0658d44.png)
+The developers created this app to bond with their love of music. We hope that users will enjoy listening to their favorite music and learn other songs they can tune into 😃 🎵🎵🎵
+
+## Features 🎵🎵🎵
+
+- Create and edit an account
+- Create, edit and delete a game rooms. Game rooms created are accessible to other registered users.
+- Search tracks and add those tracks in the game room. 🎵
```

Comment:
Search tracks which connected to Apple's iTunes Search API and add those tracks in the game room.

Reply:
Please see commit 7cf06e2 to resolve this 😃